code/src/components/Album.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Artist } from './Artist'
+
+export const Album = (props) => {
```

Comment:
When your album title is longer then 1 line it makes your album photo break the grid on top. Your photos should be in line and your text areas should be ready for longer text. 
![image](https://user-images.githubusercontent.com/50926159/111635072-69679500-87f7-11eb-83fc-87d48cc33b3b.png)